tests/settings.py

```diff
@@ -80,3 +80,6 @@
 # Wagtail settings
 
 WAGTAIL_SITE_NAME = "Wagtail Live Test Suite"
+
+LIVE_APP = "testapp"
+LIVE_PAGE_MODEL = "BlogPage"
```

Comment:
In my experience it is a common pattern to use the full package name as a prefix for settings.

`LIVE_APP` -> `WAGTAIL_LIVE_APP` and `LIVE_PAGE_MODEL` -> `WAGTAIL_LIVE_PAGE_MODEL`.

Why doe we split these settings? Do we ever need to now only the app or only the model? This is an example to do it in one go. It might be easier to configure.

https://github.com/wagtail/wagtail-live/pull/12/files#diff-edc2a40f7a19da1e5f95552ee93050701db1e7688ec24abed3d7058364862ad4R320-R322